src/ui/WidgetIndexing.vue

```diff
@@ -1,6 +1,5 @@
 <template>
   <BaseWidget v-if="connected && hasData" link="/indexing" :status="finished ? 'good' : 'neutral'">
-    Index status:
```

Comment:
I think some stuff from your screenshot is missing here?

And if reviewing the screenshot (which I like) I'd suggest changing it to "Indexed all messages" when `finished`